packages/ui-extensions-core/src/doist-card/card-elements.ts

```diff
@@ -103,6 +104,19 @@ export class DoistCard extends ContainerWithNoItems {
 
         return result
     }
+
+    static fromWithItems<T extends DoistCard>(
```

Comment:
Is this meant to be used as in:
```
       DoistCard.fromWithItems({
            items: [
                TextBlock.from({
                    id: 'some-id',
                    text: 'some text',
                    size: 'large',
                }),
            ],
        })
```
?

The code works as expected (the adaptive card that gets displayed looks as expected), but I'm getting a few IDE complains so maybe I'm not using it properly:

<img width="813" alt="Screenshot 2022-11-08 at 11 31 32" src="https://user-images.githubusercontent.com/1509326/200541757-be63e98f-e3db-42ee-a8a2-9c3cc4818c88.png">

Reply:
Reload your dev window, that will cure it.